lr_face/data_providers.py

```diff
@@ -0,0 +1,146 @@
+import csv
+from dataclasses import dataclass
+from typing import Tuple
+import os
+
+import pandas as pd
+import cv2
+import numpy as np
+from sklearn.model_selection import GroupShuffleSplit
+
+
+@dataclass
+class Images:
+    y_train: np.ndarray
+    X_train: np.ndarray
+    y_calibrate: np.ndarray
+    X_calibrate: np.ndarray
+    y_test: np.ndarray
+    X_test: np.ndarray
+
+
+def test_data(resolution):
+    """
+    return some random numbers in the right structure to test the pipeline with
+    """
+    return np.random.random([11, resolution[0], resolution[1], 3]), np.array([1, 1, 1, 2, 2, 2, 3, 3, 4, 4, 5])
+
+
+def enfsi_data(resolution, year) -> Tuple[np.ndarray, np.ndarray]:
+    folder = os.path.join('resources', 'enfsi', str(year))
+    X = []
+    y = []
+    i_cls = 1 # start on 1 as ENFSI does it
+    df = pd.read_csv(os.path.join(folder, 'truth.csv')).set_index('id')
+    for file in os.listdir(folder):
+        if not file.endswith('csv'):
+            # TODO check RGB/GRB ordering/colous usage on models.
```

Comment:
Ook width/height moeten we in de in de gaten houden, dat wil per package ook nog wel eens verschillen.